specification/ai/Azure.AI.Projects/connections/model.tsp

```diff
@@ -148,6 +158,9 @@ enum ConnectionType {
 
   @doc("Generic connection that uses API Key authentication")
   APIKey: "ApiKey",
+
+  @doc("Generic connection that uses Custom Keys authentication")
+  CustomKeys: "CustomKeys",
```

Comment:
Started a chat with Glenn and Neehar asking why this is different than "ApiKey".